markdown/en/api/ff-paging.api.md

```diff
@@ -3,42 +3,42 @@ ___
 ### Properties
 | Name | Description |
 | ---- | ----------- |
-|**paging** (Object) (default:empty)| The paging data from FACT-Finder.|
-|**show-only** (String) **Options**: true, false (default: false)| Determines if pages should only be displayed but not clickable.|
+| **paging** (Object) (default:empty) | The paging data from FACT-Finder. |
+| **show-only** (String) **Options**: "true", "false" (default: "false") | Determines if pages should only be displayed but not be clickable. |
 
 
 ### Events
 | Name | Description |
 | ---- | ----------- |
-|**dom-updated**|  This event is triggered when the element has received new data and the template for the element and all sub elements was punched out.|
+| **dom-updated** | This event is triggered when the element has received new data and the template for the element and all sub elements was punched out.|
 
 ## `ff-paging-set`
 ___
 ### Properties
 | Name | Description |
 | ---- | ----------- |
-|**state** (String) (default:all)| This property sets the condition under which the `ff-paging-set` element will become _aktive_. This is achieved by setting the the numbers behind `currentPage` - for the currently selected page - and `pageCount` - for the amount of pages - through any calculation to the desired value. The condition is then based on this value(e.g. `currentPage > 3 &&pageCount-currentPage >= 3`).|
-|**is-active** (String) **Options**: true, false (default:false)| Determines if the paging set is currently active.|
-|**paging** |(Object) (default:empty) The data from FACT-Finder.|
+| **state** (String) (default: "all") | This property sets the condition under which the `ff-paging-set` element will become _active_. This is achieved by setting the the numbers behind `currentPage` - for the currently selected page - and `pageCount` - for the amount of pages - through any calculation to the desired value. The condition is then based on this value(e.g. `currentPage > 3 &&pageCount-currentPage >= 3`). |
```

Comment:
This is achieved by setting ~~the~~ the numbers behind `currentPage` - for the currently selected page - and `pageCount` - for the amount of pages - through any calculation to the desired value. The condition is then based on this value _MISSING SPACE_ (e.g. `currentPage > 3 && _MISSING SPACE_ pageCount _MISSING SPACE_ - _MISSING SPACE_ currentPage >= 3`). |

Reply:
![image](https://user-images.githubusercontent.com/27375330/47099930-a4d09480-d236-11e8-9fdc-d9fe7282fde7.png)
Github took my comment to heart and even made it a feature. Who'd have thought?